### PR TITLE
Live chat bot

### PR DESCRIPTION
**Pull request by basil-kawak**
PTAL

### CODE REVIEW

File: src/containers/LiveChat.jsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
+import React from "react";
+import ChatBot from "react-simple-chatbot";
+import { ThemeProvider } from "styled-components";
+
+const LiveChat = () => {
+  // all available props
```
Comment (louisrli) [inline src/containers/LiveChat.jsx]: unnecessary comment

File: src/containers/LiveChat.jsx [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    headerFontColor: "#fff",
+    headerFontSize: "15px",
+    botBubbleColor: "#539be5",
+    botFontColor: "#fff",
+    userBubbleColor: "#fff",
+    userFontColor: "#4a4a4a",
+    floating: true,
+  };
+
+  const steps = [
```
Comment (louisrli) [inline src/containers/LiveChat.jsx]: for these two variables `steps` and `theme`, when they are constant like this, we normally name them in CONSTANT_CASE like this and move it outside of the component. this makes it clear to readers that this doesn't depend on anything in the function and is static. let's do that here for these configurations

**Review by louisrli — APPROVED** [review]
otherwise good

**Review by gbilgedal — APPROVED** [review]
lgtm, after louis' requests.

**Pull request merged**